mobify/platform-scaffold
# [Integration Manager] JSDoc for commands

This PR introduces a npm script to generate the documentation for Integration Manager commands. It uses [JSDoc](usejsdoc.org) to extract doc comments. 

 **JIRA**: [**WEBDATA-17** Finalize Command API](https://mobify.atlassian.net/browse/WEBDATA-17)
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Add npm script to extra IM doc comments
- Add README used when generating docs 
- Add README used to describe the IM folder

## How to test-drive this PR
- `npm install` (there are new package.json dependencies)
- Run `npm run im:docs && open app/integration-manager/docs/index.html`
- Review some of the command documentation (just spot check a few)
